test/pipeline/pipeline_test_utils.py

```diff
@@ -43,12 +45,13 @@ def process_file(input_file, arch, target="xcore"):
     output_file = os.path.join(pipeline_output_base_dir + "_" + arch + "_" + target, wav_name)
 
     pipeline_bin = pipeline_bins[arch][target]
+    stdout = ""
```

Comment:
This line isn't necessary because `stdout` will always be set by the if...else below.